### PR TITLE
SunnyValley.repo code clean

### PR DESCRIPTION
**Pull request by ureyni**
We want to release the changes in version 23.7.12

### CODE REVIEW

File: vendor/sunnyvalley/src/opnsense/scripts/firmware/repos/SunnyValley.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+$fileContents = file_get_contents($conf . '.shadow');
+
+if (file_exists('/usr/local/zenarmor/bin/eastpect')) {
+    $uuid = shell_safe('/usr/local/zenarmor/bin/eastpect -s');
+    if ($uuid != '') {
+        $fileContents = str_replace('/latest"', '/' . $uuid . '"', $fileContents);
+    }
+}
+
+file_put_contents($conf, $fileContents);
```
Comment (fichtner) [inline vendor/sunnyvalley/src/opnsense/scripts/firmware/repos/SunnyValley.php:47]:
```suggestion
file_put_contents($conf, $fileContents);

```

**Comment by fichtner** [comment]
@ureyni can you merge from master? You branch isn't up to date...

**Comment by ureyni** [comment]
@fichtner  I merged with master.

**Comment by fichtner** [comment]
All done, thanks!

**Pull request closed without merging**